src/Tabs/Tabs.js

```diff
@@ -4,26 +4,47 @@ import React, { Component, PropTypes, Children, cloneElement } from 'react';
 import classNames from 'classnames';
 import { createStyleSheet } from 'jss-theme-reactor';
 import EventListener from 'react-event-listener';
+import ScrollbarSize from 'react-scrollbar-size';
+import scroll from 'scroll';
+import { throttle } from 'lodash';
```

Comment:
@tassinari Indeed, and you did well. I also would have set the flag to true before calling the function, as a recursive call would lead to an infinite loop.